sdk/eventhub/azure-eventhub/azure/eventhub/_common.py

```diff
@@ -328,7 +339,7 @@ def body(self):
         For :class:`azure.eventhub.amqp.AmqpMessageBodyType.VALUE<azure.eventhub.amqp.AmqpMessageBodyType.VALUE>`,
         the body could be any type.
 
-        :rtype: Any
+        :rtype: int or float or bytes or str or Dict or List or uuid.UUID
```

Comment:
bool is missing, also connecting by str looks a bit weird.. should it be like the following for sphinx:
`int, float, bytes, str, Dict, List or uuid.UUID`?

Reply:
good catch!

I think they should each be or b/c that's how I've seen it in other places. For ex: https://github.com/Azure/azure-sdk-for-python/blob/60a7f629d56a8380806df322d1d5a3fa1559b718/sdk/servicebus/azure-servicebus/azure/servicebus/aio/_servicebus_client_async.py#L48

Reply:
looks like you're right:

<img width="436" alt="image" src="https://user-images.githubusercontent.com/47871814/149037826-b7bfa756-d5f5-4ee9-8346-447978ed2fbb.png">


https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html

Reply:
but one more thing, does sphinx support `Dict` or `List` which I feel are more of a "typing" thing as Laurent pointed out in the issue:https://github.com/Azure/azure-sdk-for-python/issues/22051

probably should just be `dict` or `list` (lower case)

Reply:
ah yep!